backend/src/main/java/atlantbh/restaurants/exceptions/StringMissmatchException.java

```diff
@@ -0,0 +1,12 @@
+package atlantbh.restaurants.exceptions;
+
+public class StringMissmatchException extends Exception {
```

Comment:
Maybe you should add some javadoc to these classes, so it would be easier to know when you need to call this exception..

Reply:
Added javadoc in 8c841a20e489a31c66d177c8682339f9d4f19de0